terragraph_planner/data/template.yaml

```diff
@@ -1,106 +1,104 @@
+  # more details.
   LOGGER_LEVEL: INFO
 
-  # [Optional][String] The file path of the log file. If it is null, the planner
-  # will not give the log in the file.
+  # [Optional][String] The file path of the log file. If it is null, log will
+  # not be recorded to file.
   LOG_FILE: null
 
-  # [Bool] Whether or not the planner prints log to the stderr.
+  # [Bool] Enable logging to stderr.
   LOG_TO_STDERR: True
 
 DATA:
-  # [String] The area that planning will be restricted to. The file
-  # can contain only one polygon. File type: KML or KMZ
+  # [Required][String] The area of interest of the network plan. File type:
+  # KML/KMZ.
   BOUNDARY_POLYGON_FILE_PATH: boundary.kml
 
-  # [List of String] A list of GeoTIFF files containing
-  # the height of canopy (terrain elevation + height of any clutter above
-  # ground) (.tif).
-
+  # [List of String] A list of GeoTIFF files containing the height of canopy
+  # (terrain elevation + height of any clutter above ground) File type: TIF.
   DSM_FILE_PATHS:
     - dsm.tif
 
-  # [String] DTM_FILE_PATH contains the digital terrain
-  # model, which represents the elevation of the earth's surface. File type: TIF
-  # DHM_FILE_PATH contains the digital height model, which represents the height
-  # of aboveground objects. File type: TIF
-
+  # [String] DTM_FILE_PATH contains the digital terrain model, which represents
+  # the elevation of the earth's surface. File type: TIF.
+  # DHM_FILE_PATH contains the digital height model, which represents the
+  # height of aboveground objects. File type: TIF.
   DTM_FILE_PATH: dtm.tif
   DHM_FILE_PATH: dhm.tif
 
-  # [String] The files that contain the geometric locations and
-  # attributes of geographic features, such as polygons and heights of
-  # buildings. Get data from a zipped shape file or a KML/KMZ file.
-  # If a zipped shape file is given, They must be uploaded as a ZIP file containing
-  # SHP, SHX, PRJ and DBF shapefiles that are not in a nested directory. For
-  # example, the directory for the SHP file should follow "shapefile.zip/
-  # shp_file.shp" and not "shapefile.zip/some_folder/shp_file.shp."
-
+  # [String] The files that contain the geometric locations and attributes of
+  # geographic features, such as polygons and heights of buildings. File type:
+  # zipped shape file or a KML/KMZ file. If a zipped shape file is given, it
+  # must be uploaded as a ZIP file containing SHP, SHX, PRJ and DBF shapefiles
+  # that are not in a nested directory. For example, the directory for the SHP
+  # file should follow "shapefile.zip/shp_file.shp" and not
+  # "shapefile.zip/some_folder/shp_file.shp."
   BUILDING_OUTLINE_FILE_PATH: buildings.zip
 
-  # [String] Path of KML, KMZ or CSV file containing the potential POP/DN/CN
-  # sites. KML or KMZ file can also contain a list of exclusion zones.
+  # [String] Path of the file containing the candidate POP/DN/CN sites. File
+  # type: KML/KMZ or CSV. KML/KMZ files can also contain a list of exclusion
```

Comment:
Does this imply CSV cannot contain exclusion zones? If so, I think it's better to say "KML/KMZ files can contain a list of exclusion, and CSV files cannot" or something like that. (Or if this is already explained somewhere else, you can point to that or just remove this sentence.)